contracts/src/Teleporter/ITeleporterMessenger.sol

```diff
@@ -35,11 +35,11 @@ struct TeleporterFeeInfo {
 }
 
 /**
- * @dev Interface that describes functionalities for a cross chain messenger.
+ * @dev Interface that describes functionalities for a cross-chain messenger implementing the Teleporter protcol.
```

Comment:
we have some references to "Teleporter Messenger protocol", should unify these

Reply:
Here I think just calling it Teleporter is more appropriate. "Teleporter Messenger" is an implementation of the Teleporter protocol. This interface is specifically tied to "Teleporter Messenger", but I think we should consider paring it back to conform only the Teleporter protocol. All the additional methods included in `TeleporterMessenger.sol` would be outside of the protocol.

Reply:
do you think it makes sense that when we refer to "protocol" it refers to Teleporter protocol only? Like you said Teleporter Messenger is an implementation of the protocol, so in the READMEs we can remove all references to Teleproter Messenger as a protocol.

Reply:
Good call. Replaced "Teleporter Messenger protocol" with "Teleporter protocol" in a few places. Lmk if you spot any others.